jetty-servlet/src/main/java/org/eclipse/jetty/servlet/ServletHolder.java

```diff
@@ -1284,14 +1285,15 @@ public void service(ServletRequest req, ServletResponse res) throws ServletExcep
      * @throws IllegalAccessException if not allowed to create a new instance
      * @throws InstantiationException if creating new instance resulted in error
      */
-    protected Servlet newInstance() throws ServletException, IllegalAccessException, InstantiationException
+    protected Servlet newInstance() throws ServletException, IllegalAccessException, InstantiationException,
+        NoSuchMethodException, InvocationTargetException
```

Comment:
Adding more exception to the signature is not enough.
You should A) update the javadocs too; B) update the catch block below, where `ServletException` is unwrapped.